backend/schema/validate.go

```diff
@@ -611,6 +640,44 @@ func validateIngressRequestOrResponse(scopes Scopes, module *Module, n *Verb, re
 	return
 }
 
+func validateVerbSubscriptions(module *Module, v *Verb, md *MetadataSubscriber, scopes Scopes, schema optional.Option[*Schema]) (merr []error) {
+	merr = []error{}
+	var subscription *Subscription
+	for _, decl := range module.Decls {
```

Comment:
I am seeing this pattern all the time. Let's write a helper for extracting an impl from a slice of sum types. Something like:

```go
func FindVariant[T any, U any](slice []U) (T, bool) {
	for _, el := range slice {
		if found, ok := any(el).(T); ok {
			return found, true
		}
	}
	var zero T
	return zero, false
}

// subscription, ok := FindVariant[*Subscription](module.Decls)
```